ngraph_bridge/ngraph_encapsulate_op.cc

```diff
@@ -516,7 +538,7 @@ void NGraphEncapsulateOp::ComputeUsingParallelExecutor(OpKernelContext* ctx) {
 
   // Now return them to the cache
   ngraph::Event event_return_tensor("Return Tensor", "", "");
-  m_parallel_executor->ReturnPipelinedTensors(ng_exec, get<0>(io_tensors));
+  pipelined_tensor_store->return_tensors(get<0>(io_tensors));
```

Comment:
m_parallel_executor->ReturnPipelinedTensors used to be guarded by the mutex, which is not required now. We created using the cache's lock. Each thread will get a different id and then can return it without the mutex. Is that the right understanding?

Reply:
I don't think its needs to be locked, Got rid of ReturnPipelinedTensor(), pts inside the cache is locked and safe, outside it is local to the thread, so do not need protection.